src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A printable interface for the particulars of a Person in the addressbook.
+ * Implementations should guarantee: a printable string representation of the object.
+ */
+public interface Printable {
+    String getPrintableString();
```

Comment:
Header comment missing here. Use Returns... as a description and omit @return line.